src/main/java/org/opentripplanner/standalone/config/CommandLineParameters.java

```diff
@@ -98,6 +98,14 @@
                     + "interfaces.")
     public String bindAddress = DEFAULT_BIND_ADDRESS;
 
+    @Parameter(names = {"--clientPath"},
+            description = "Path to serve local client files at.")
+    public String clientPath = null;
+
+    @Parameter(names = {"--disableNativeClient"},
```

Comment:
Would you be open renaming it to "debug client", which is the name we use in the dev team?

Native client to me means something like C++ or iOS.

Reply:
Sure, I'm ambivalent on the naming. I'll change.

Reply:
Change complete